### DISCUSSION TITLE
Shortest Dependency Path between Nouns

### DISCUSSION THREAD

**Discussion by priamai** [explosion/spaCy#13015] [Help: Coding & Implementations] [answered]
Hi!
I would like to see how to implement a shortest dependency path between entities in Spacy.
I couldn't find anything on the documentation or other projects but just this example online: 
https://towardsdatascience.com/how-to-find-shortest-dependency-path-with-spacy-and-stanfordnlp-539d45d28239
Unfortunately I don't have a paid subscription to Medium.
Is there an official implementation of that method?
It is mentioned in several NLP papers so would be nice to have a vanilla utility in Spacy.
Cheers.

**Comment by shadeMe** [comment, marked as answer]
We do not provide an official implementation. 

As far as I can tell, one doesn't need a paid subscription to view the article - just a free account. In any event, it's also accessible through [this link](https://archive.ph/mRINO).

**Comment by priamai** [comment]
Hello,
I am getting hit by that paywall, but thanks for the workaround.
Yes the implementation seems trivial, I can easily provide a utility for spacy to have that functionality.
I am guessing we don't want to have a dependency from networkx?

**Reply by shadeMe** [reply]
[spaCy Universe](https://spacy.io/universe) would be a good place for such a resource. Feel free to open a PR in the spaCy repo if you'd like to share your implementation.
